src/api/instance/instance.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import {type CredrankInput} from "../credrank";
+import {CredGraph} from "../../core/credrank/credGraph";
+import {Ledger} from "../../core/ledger/ledger";
+import {type DependenciesConfig} from "../dependenciesConfig";
+
+/**
+This interface represents a SourceCred Instance, which is simply a read/write
+interface for configurations and outputs of the main SourceCred API.
```

Comment:
nits on effective documentation:

> This interface represents a SourceCred Instance, which is simply a read/write
> interface for configurations and outputs of the main SourceCred API.

“This interface” is superfluous; the doc comment for this interface
clearly describes this interface.

“represents a SourceCred Instance, which is” is superfluous; it’s in the
SourceCred repository and is called `Instance`.

“simply a read/write interface for […]” is meaningful!

“configurations and outputs” makes me ask, “the natural dual of
*outputs* is *inputs*; in what sense are *configurations* different?”.
Since the method does in fact return just a `CredrankInput`, perhaps
“inputs” is simpler.

“of the main SourceCred API” feels helpful to me, because it specifies
the level of abstraction that I should expect. But it looks like this
really is all about CredRank. Maybe we could say that more precisely.

So, how about:

```javascript
/**
 * Simple read-write interface for CredRank inputs and outputs.
 */
```

?

Exercise for the reader: apply this same philosophy to `LocalInstance`.

Reply:
I'll apply most this feedback, though I don't want to narrow the scope to just credrank, since the plan is to incrementally add the inputs and outputs for other APIs. (I could stub them, but would rather leave them out for now.)